test-integration/src/test/java/nl/nn/adapterframework/filesystem/SelfContainedBasicFileSystemTest.java

```diff
@@ -59,30 +59,30 @@ public void testFiles()  throws Exception{
 				displayFile(it.next());
 				fail("just created folder ["+folderName+"] should be emtpty");
 			}
-			assertFalse("just created folder ["+folderName+"] should be emtpty", it!=null && it.hasNext());
+			assertFalse(it!=null && it.hasNext(), "just created folder ["+folderName+"] should be emtpty");
 		}
 		F sourceFile = null;
 		F destFile1 = null;
 		try(DirectoryStream<F> ds = fileSystem.listFiles(sourceOfMessages_folder)) {
 			Iterator<F> it = ds.iterator();
-			assertTrue("there must be at least one messsage in the sourceOfMessages_folder ["+sourceOfMessages_folder+"]", it!=null && it.hasNext());
+			assertTrue(it!=null && it.hasNext(), "there must be at least one messsage in the sourceOfMessages_folder ["+sourceOfMessages_folder+"]");
 
 			sourceFile =  it.next();
-			assertTrue("file retrieved from folder should exist", fileSystem.exists(sourceFile));
-			//assertFalse("name of source file should not appear in just created folder", fileSystem.filenameExistsInFolder(folderName, fileSystem.getName(sourceFile)));
+			assertTrue(fileSystem.exists(sourceFile), "file retrieved from folder should exist");
+			//assertFalse(fileSystem.filenameExistsInFolder(folderName, fileSystem.getName(sourceFile)), "name of source file should not appear in just created folder");
```

Comment:
Why do we keep these commented-out assertions? (Here, and below). Shouldn't we just remove those?